lib/openshift/horizontal_pod_autoscaler.rb

```diff
@@ -21,5 +21,29 @@ def current_cpu_utilization_percentage(user: nil, cached: true, quiet: false)
         dig('status', 'currentCPUUtilizationPercentage').
         to_i # it can be nil when zero thus using #to_i
     end
+
+#    def target_average_value(user: nil, cached: true, quiet: false)
+#      obj = raw_resource(user: user, cached: cached, quiet: quiet)
+#      return obj.dig('spec', 'targetAverageValue')
+#    end
+
+#    def current_average_value(user: nil, cached: true, quiet: false)
+#      raw_resource(user: user, cached: cached, quiet: quiet).
+#        dig('status', 'currentAverageValue').
+#        to_i # it can be nil when zero thus using #to_i
+#    end
+#    
+#
+    def target_average_utilization(user: nil, cached: true, quiet: false)
+      obj = raw_resource(user: user, cached: cached, quiet: quiet)
+      return obj.dig('spec', 'targetAverageUtilization')
+    end
+
+    def current_average_utilization(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).
+        dig('status', 'currentAverageUtilization').
+        to_i # it can be nil when zero thus using #to_i
+    end
+
```

Comment:
The two methods: `target_average_utilization` returns a string or nil, `current_average_utilization` returns integer. Judging from their names, they should return the same data type. Can you let `target_average_utilization` return an integer as well?